src/DynamoCore/Configuration/IPreferences.cs

```diff
@@ -124,4 +138,29 @@ public interface IPreferences
         /// otherwise.</returns>
         bool Save(string filePath);
     }
+
+    /// <summary>
+    /// Represents data about active state of preview background
+    /// </summary>
+    public class BackgroundPreviewActiveState
```

Comment:
I feel like this requires marking with some XML attribute so that deserialization occurs correctly since this class is stored inside a list... does this *just* work?